components/library/libspf2/Makefile

```diff
@@ -0,0 +1,40 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Friedrich Kink
+#
+
+BUILD_BITS=		64 # for binaries or 32_and_64 for libraries
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=         libspf2
+COMPONENT_VERSION=      1.2.10
+COMPONENT_SUMMARY=      libspf2 implements the Sender Policy Framework, a part of the SPF/SRS protocol pair.
+COMPONENT_PROJECT_URL=  https://www.$(COMPONENT_NAME).org
+COMPONENT_FMRI=         library/libspf2
+COMPONENT_CLASSIFICATION=Applications/Internet
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
```

Comment:
COMPONENT_CLASSIFICATION looks a little bit strange for a library.

Reply:
I hope System/Libraries looks better.